### PR TITLE
v18.0

### PR DESCRIPTION
**Pull request by qq7**

### CODE REVIEW

**Review by OnGle — APPROVED** [review]
lgtm

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 WEBMIN_FW_TCP_INCOMING = 22 80 443 12320 12321
```
Comment (Christie018) [inline Makefile]: I believe port 12320 can be removed for non-existent web shell.

**Review by Christie018 — CHANGES_REQUESTED** [review]
I've reviewed the code changes and requested changes.

This PR is dependent on common PR#279:
https://github.com/turnkeylinux/common/pull/279

**Comment by Christie018** [comment]
LGTM!

**Comment by JedMeister** [comment]
Closed as per https://github.com/turnkeylinux/tracker/issues/1982

**Pull request closed without merging**